apps/landingPage/src/components/Headline/Subtitle.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import Icons from './Icons';
+import Text from './Text';
+import { BREAKPOINTS } from '../../mediaQueriesConfig';
+
+const bodyText =
+  "You've had a look at Tequila, but you've always preffered your Tequila mixed with something else. Just grab a Margarita.";
+
+type Props = {|
+  +align: boolean,
+|};
+
+export default function Subtitle(align: Props) {
+  return (
+    <Container>
+      <HeaderAndDesignWrapper>
+        <BackgroundGradientAccent>
+          <GreenRectangle />
+        </BackgroundGradientAccent>
+        <Text align={!align} type="title2" text={bodyText} />
```

Comment:
Maybe we should keep it similar with Text from margarita and have text as a children

Reply:
Changed `prop` to children and also renamed component to the `TextHeadline`

Reply:
this looks weird `align={!align}`

Reply:
is `align={align}` less weird? as I just changed it to that

Reply:
Yes, it is better 😄